src/tensorwaves/data/tf_phasespace.py

```diff
@@ -25,21 +28,31 @@ def __init__(
         )
 
     def generate(
-        self, size: int, random_generator: UniformRealNumberGenerator
+        self,
+        size: int,
+        seed_or_rng: Optional[Union[UniformRealNumberGenerator, int]] = None,
```

Comment:
Naming may be approved: currently sounds more like a boolean.

Reply:
I believe the pythonic thing would be to split it up into two arguments, one seed one rng, and rng takes priority over seed.